src/lib/models/workflow-execution.ts

```diff
@@ -19,6 +19,11 @@ export const toWorkflowExecution = (
   const historyEvents = response.workflowExecutionInfo.historyLength;
   const url = `/workflows/${id}/${runId}`;
   const taskQueue = response?.executionConfig?.taskQueue?.name;
+  const parentNamespaceId = response?.workflowExecutionInfo?.parentNamespaceId;
+  const parent = response?.workflowExecutionInfo?.parentExecution;
+  const stateTransitionCount =
+    response?.workflowExecutionInfo?.stateTransitionCount;
+
```

Comment:
We might want to double check this one with `strict: true`

Reply:
Switched to response.workflowExecutionInfo.stateTransitionCount